MELA/src/TUtil.cc

```diff
@@ -1715,6 +1715,8 @@ double TUtil::InterpretScaleScheme(const TVar::Production& production, const TVa
         production == TVar::ttH
         ||
         production == TVar::bbH
+        ||
+        production == TVar::qqH
```

Comment:
implement the same scaling as heavy quarks for light quarks (because why not I guess).